recipe/meta.yaml

```
@@ -26,17 +32,85 @@ requirements:
     - make  # [unix]
   host:
 
+outputs:
+  # zlib-ng gets built when compat is NOT enabled; versioned with the regular zlib-ng release
+  - name: zlib-ng
+    build:
+      skip: true  # [compat == "yes"]
+  # zlib and libzlib get built when compat is enabled; they are versioned after the zlib API
+  - name: libzlib
+    version: {{ zlib_api }}
+    build:
+      skip: true  # [compat == "no"]
+      string: h{{ PKG_HASH }}_ng_{{ version.replace(".", "") }}_{{ number }}  # [compat == "yes"]
+      run_exports:
+        # This is for compat mode, just libzlib, with major pinning
+        - {{ pin_subpackage('libzlib', max_pin='x') }}  # [compat == "yes"]
+      track_features:  # [compat == "yes"]
+        - zlib-ng-compat  # [compat == "yes"]
```

```suggestion
      string: h{{ PKG_HASH }}_ng_{{ version.replace(".", "") }}_{{ number }}
      run_exports:
        # This is for compat mode, just libzlib, with major pinning
        - {{ pin_subpackage('libzlib', max_pin='x') }}
      track_features:
        - zlib-ng-compat
```

We don't need these, do we?

---

I guess so, no clue why they were added there xP.